src/escolasegura.py

```diff
@@ -9,7 +9,7 @@
 import utils
 import pages.simulationContainer as sc
 
-st.set_page_config(layout="wide")
+st.set_page_config(page_title="Escola Segura", page_icon=None,layout='wide', initial_sidebar_state='collapsed')

```

Comment:
esse `initial_sidebar_state ` é o mesmo do hack abaixo? se for acho que vale deixar um só ne? e ícone!! da para adicionarmos a logo??

Reply:
Que hack de baixo? E sim dá pra voce mudar o logo para o que quiser, só não fiz pois o dashboard está me tomando um tempo, fique a vontade.

Reply:
update aqui:

![image](https://user-images.githubusercontent.com/20743819/100151586-1c466880-2e80-11eb-8146-09bba558ee60.png)